mne/beamformer/tests/test_dics.py

```diff
@@ -270,7 +273,11 @@ def test_apply_dics_csd():
     # Test using a real-valued filter
     filters_real = make_dics(epochs.info, fwd_surf, csd, label=label, reg=reg,
                              real_filter=True)
-    power, f = apply_dics_csd(csd, filters_real)
+    # Also test whether stc contains src:
+    with warnings.catch_warnings(record=True) as wrn:
+        power, f = apply_dics_csd(csd, filters_real)
+    assert not any('src should not be None' in str(ww.message) for ww in wrn)
```

Comment:
If is this really meant to assert that there are no warnings, then a simple `assert len(w) == 0` is better. If there are other warnings that are caught by this, please add a short comment saying what they are or assert what they are with `assert any('...' in str...)`

Reply:
I am trying to see if the specific warning "src should not be None [...]" that would be thrown if  ``src_type`` are not supplied to ``_make_stc`` is fetched. It was implemented by @agramfort during the fix #5231 for the issue #5224

Reply:
My check is still checks for that, plus other warnings so it's more general